container/typings/LuigiContainer.svelte.d.ts

```diff
@@ -73,6 +73,12 @@ export default class LuigiContainer extends HTMLElement {
    */
   nodeParams: Object;
 
+  /**
+   * If set to true, luigi container webcomponent don't have a shadow dom.
```

Comment:
>  * If set to true, luigi container webcomponent don't have a shadow dom.

 Maybe sth like: 
 _If set to true, luigi container webcomponent will not be rendered in a shadow DOM_
 update the docu part also, guess its done pre-commit anyway